### PR TITLE
Document openSUSE Leap Micro, remove openSUSE MicroOS

### PR DESCRIPTION
**Pull request by juliogonzalez**
# Description

Document openSUSE Leap Micro, remove openSUSE MicroOS

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3 (only later, when support is added!)
- [ ] Manager-4.2

# Links

https://github.com/uyuni-project/uyuni/pull/6550 and https://github.com/SUSE/spacewalk/issues/20357

### CODE REVIEW

**Comment by juliogonzalez** [comment]
You can check the renders at https://w3.suse.de/~jgonzalez/opensuseleapmicro/

**Comment by juliogonzalez** [comment]
DO NOT MERGE THIS UNLESS INSTRUCTED BY  @juliogonzalez, AS WE HAVE A BUGFIX PENDING!!!

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc
Comment (keichwa) [inline modules/client-configuration/pages/clients-opensuseleapmicro.adoc:14]:
```suggestion
They do not affect the running system because no changes are activated until the system is rebooted.
```

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc
Comment (keichwa) [inline modules/client-configuration/pages/clients-opensuseleapmicro.adoc:109]:
```suggestion
{leapmicro} clients require reboot after registering.
```

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc
Comment (keichwa) [inline modules/client-configuration/pages/clients-opensuseleapmicro.adoc:115]:
```suggestion
To register {leapmicro} clients, you need a bootstrap repository.
```

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc
Comment (keichwa) [inline modules/client-configuration/pages/clients-opensuseleapmicro.adoc:117]:
```suggestion
You can manually create the bootstrap repository from the command prompt, using the command:
```

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc
Comment (keichwa) [inline modules/client-configuration/pages/clients-opensuseleapmicro.adoc:123]:
```suggestion
For more information on registering clients, see xref:client-configuration:registration-overview.adoc[].
```

**Review by keichwa — APPROVED** [review]
cosmetic change proposals.

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc
Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-opensuseleapmicro.adoc:14]: Fine, but keep in mind this is a copy paste of what you already have for SLE Micro :-)

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc
Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-opensuseleapmicro.adoc:117]: Same as before.

File: modules/client-configuration/pages/clients-opensuseleapmicro.adoc
Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-opensuseleapmicro.adoc:123]: Same as before.

**Comment by juliogonzalez** [comment]
This PR can be merged. I verified the bootstrap issues are gone after @wweellddeerr fixes for https://github.com/SUSE/spacewalk/issues/20349

**Review by jcayouette — APPROVED** [review]

**Pull request merged**
